build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.1'
+        classpath 'com.android.tools.build:gradle:4.0.0-beta04'
```

Comment:
Is this needed?

Reply:
What I have to say is that I use Android Studio Version 4.0 Beta 4 as build environment. And every time I port an old project into the IDE it recommends to update the Gradle plugin(4.0.0-beta04) and the Gradle Version (6.1.1). If you have that problem it should be considered. I tried to reproduce the same error without success.
But if you run Android Studio 3.5 you can try to run Gradle Plugin 3.5.0 and Gradle Version 5.4.1 because it is recommended by the IDE with that version, it might help.

Reply:
Android Studio version is irrelevant, as the project isn't dependent on it. 

Changing the Gradle version to a beta version should have a valid justification.

And are you able to build the project? Not from Android Studio, directly with gradle?

Reply:
Well, i never tried to build outside of Android Studio. Is there a reason to build it outside? I thought the Build Prozess is completly covered by Android Studio.

Reply:
No project should be tied to a specific editor/IDE. Which editor/IDE someone uses is up to them, it should never be specified in the project.

You need to specify a build system, though, and that's gradle (standalone).

PS. I also use Android Studio, but the build isn't and will never be dependent on it.

Reply:
Okay. But I can only confirm that the Build process in Android Studio works at least in my environment flawlessly.